src/rpc/client.cpp

```diff
@@ -127,6 +127,8 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "testmempoolaccept", 0, "rawtxs" },
     { "testmempoolaccept", 1, "maxfeerate" },
     { "submitpackage", 0, "package" },
+    { "submitpackage", 1, "maxfeerate" },
+    { "submitpackage", 2, "maxburnamount" },
```

Comment:
Should enable passing `maxfeerate` and `maxburnamount` as a config variables since it is used by various RPC, downstream might prefer this?

Reply:
startup config? Maybe? Could open an issue

Reply:
Done in https://github.com/bitcoin/bitcoin/issues/29217

Reply:
Note  a good idea, because we have to restart whenever we want to update the threshold for an RPC call, can be resolved.